package/gluon-config-mode-contact-info/README.md

```diff
@@ -0,0 +1,8 @@
+gluon-config-mode-contact-info
```

Comment:
Please move the documentation to `docs/package/` and try to use a similar style like the existing ones.